x/multi-staking/keeper/keeper_test.go

```diff
@@ -0,0 +1,42 @@
+package keeper_test
+
+import (
+	"github.com/stretchr/testify/suite"
+	"testing"
+
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/cosmos/cosmos-sdk/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/realio-tech/multi-staking-module/testing/simapp"
+	multistakingkeeper "github.com/realio-tech/multi-staking-module/x/multi-staking/keeper"
+	multistakingtypes "github.com/realio-tech/multi-staking-module/x/multi-staking/types"
+)
+
+type KeeperTestSuite struct {
+	suite.Suite
+
+	ctx      sdk.Context
+	msKeeper *multistakingkeeper.Keeper
+}
+
+func (suite *KeeperTestSuite) SetupTest() {
+	app := simapp.Setup(false)
+
+	storeKey := storetypes.NewKVStoreKey(multistakingtypes.StoreKey)
+	memKey := storetypes.NewMemoryStoreKey(multistakingtypes.MemStoreKey)
+	encCfg := simapp.MakeTestEncodingConfig()
+	testCtx := testutil.DefaultContext(storeKey, storetypes.NewTransientStoreKey("transient_test"))
```

Comment:
This should be mem store key

Reply:
I think we should mount both mem key and trasient key for avoid future conflicts. You can check my new commit here [Mount memkey to memdb](https://github.com/realiotech/multi-staking/pull/19/commits/b4cadcd19d7bca6ec39aeeb1fa0030bf6478d464)